### PR TITLE
Updates required for Summed ADC changes

### PR DESCRIPTION
**Pull request by amerega**

### CODE REVIEW

File: dunereco/HitFinderDUNE/TimeBasedDisambig.cxx [hunk 1]
```diff
@@ -470,7 +470,8 @@ void TimeBasedDisambig::RunDisambig( const std::vector< art::Ptr<recob::Hit> > &
 		       InductionAll[uv0].RMS,
 		       InductionAll[uv0].PeakAmplitude,
 		       InductionAll[uv0].SigmaPeakAmplitude,
-		       InductionAll[uv0].SummedADC,
+		       InductionAll[uv0].SummedADC,//ROI
```
Comment (tomjunk) [inline dunereco/HitFinderDUNE/TimeBasedDisambig.cxx:473]: Did this change the number or arguments?

Comment (amerega) [reply dunereco/HitFinderDUNE/TimeBasedDisambig.cxx:473]: It did not change the number of arguments for "(std::vector<HitPos> InductionAll)" but an additional argument was needed in the hit definition so "InductionAll[uv0].SummedADC" was used twice

Comment (aolivier23) [reply dunereco/HitFinderDUNE/TimeBasedDisambig.cxx:473]: @tomjunk may I merge this today?  I think I need it to build with the new LArSoft version.

**Review by aolivier23 — APPROVED** [review]
Seems like the changes to how recob::Hit is set up are OK for now.  According to https://indico.fnal.gov/event/64331/, ROIHitSummedADC is just an optional place to put the result of a different algorithm for counting hit ADC.  In e.g. TimeBasedDisambig, we're just choosing not to use another algorithm.

**Review by aolivier23 — APPROVED** [review]
In TimeBasedDisambig, we're just choosing to use the same algorithm for both ROIHitSummedADC and HitSummedADC.  https://indico.fnal.gov/event/64331/ makes it seem like we have that option.

**Pull request merged**